lib/instrumentation/modules/mysql-common.js

```diff
@@ -0,0 +1,161 @@
+'use strict'
```

Comment:
It might be better to move this file up a level, like we do with `http-shared.js`, so we aren't conflicting with the possible `mysql-common` package name.